<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationOuvrage</class>
 <widget class="QDialog" name="CreationOuvrage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreationOuvrage</string>
  </property>
  <widget class="QLabel" name="Bienvenue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pour ajouter un ouvrage , veuillez saisir les informations suivantes.</string>
   </property>
  </widget>
  <widget class="QLabel" name="NomAuteur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'auteur de l'ouvrage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitreOuvrage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre de l'ouvrage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="VilleEdition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville d'édition de l'ouvrage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Editeur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Éditeur de l'ouvrage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="AnneeEdition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Année d'édition de l'ouvrage : </string>
   </property>
  </widget>
  <widget class="QLabel" name="IdentifiantISBN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant de l'ouvrage (ISBN)  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="QuitterMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Une fois que vous avez fini de saisir toutes les informations,  veuillez cliquer sur ok.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>500</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="Fenetre">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asssistance pour ajout d'un ouvrage. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomAuteurSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TitreOuvrageSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="VilleditionSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditeurSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AnneeSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ISBNSaisi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Fermer">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer la fenetre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Fermer</sender>
   <signal>clicked()</signal>
   <receiver>CreationOuvrage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CreationOuvrage</receiver>
   <slot>Validation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Validation()</slot>
 </slots>
</ui>
